### DISCUSSION TITLE
[13주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#89] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
* [FrontEnd] Browser Rendering - reflow와 repaint #84
2. 김정윤
* [Nginx] Nginx의 개념 #88
3. 이주형
* [OS] Context switching #86
4. 이동호
* [Spring] RequestBody #90

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 🙂🌸
* 좋았던 점
* 건의사항
* 느낀점

**Reply by leeejuhyeong** [reply]
다음 OS 주제는 뭐로할까요

**Reply by ChaerinYu** [reply]
후후 오랜만에 네 명 다 모였네요

**Reply by doho-ho** [reply]
야근할게요 ㅎㅎ

**Reply by KJY97** [reply]
특화PJT 하면서 https 너무 하고 싶었는데.. 결국 못하고 Nginx 공부.. ㅎ
네트워크 어렵네요.. 모두 감사합니다..
